core/db/migrate/20210527094055_create_spree_products_stores.rb

```diff
@@ -0,0 +1,21 @@
+class CreateSpreeProductsStores < ActiveRecord::Migration[5.2]
+  def change
+    create_table :spree_products_stores, id: :serial do |t|
```

Comment:
@mad-eel we need to add a check if `spree_products_stores` does not exist already, because if someone used spree multi-domain earlier this migration will fail